components/hab/src/command/studio/enter.rs

```diff
@@ -43,6 +47,24 @@ fn set_env_var_from_config(env_var: &str, config_val: Option<String>, sensitive:
     }
 }
 
+fn cache_ssl_cert_file(cert_file: &str, cert_cache_dir: &Path) -> Result<()> {
+    let cert_path = Path::new(&cert_file);
+
+    let cert_filename = cert_path.file_name().expect("SSL_CERT_FILE is not a file");
+    let cache_file = cert_cache_dir.join(&cert_filename);
+
+    if cache_file.exists() && cert_path.exists() && is_same_file(&cache_file, &cert_path)? {
+        return Err(Error::CacheSslCertError("Source and destination certificate are the same \
+                                             file"
+                                                  .to_string()));
+    }
+
+    debug!("Caching SSL_CERT_FILE {}", cert_file);
```

Comment:
In case of errors, this would probably be more useful at the top of the function and also inclusive of the destination. Something like
```suggestion
    debug!("Caching SSL_CERT_FILE {:?} -> {:?}",
           cert_file, cert_cache_dir);
```